package.json

```diff
@@ -39,15 +39,16 @@
     "coverage": "codecov"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-0 || ^16.0.0-0",
+    "react": "^16.4.0-0",
     "redux": "^2.0.0 || ^3.0.0 || ^4.0.0-0"
   },
   "dependencies": {
     "hoist-non-react-statics": "^2.5.5",
```

Comment:
This will need to be updated to v3.

`React.forwardRef` returns an object, which, without the update to v3, will have its fields hoisted onto the component returned by `connect()`, which will in turn will 'skip' the `connect()` component itself because the hoisted fields make the parent look like the interior of the `forwardRef` invocation.

If none of that makes sense, suffice it to say that an upgrade is needed and an outdated `hoist-non-react-statics` is to blame for weird behavior at the intersection of HOCs and `forwardRef`.

Reply:
thanks, next commit updates this issue